elastic/datadog_checks/elastic/metrics.py

```diff
@@ -759,6 +759,10 @@
     'elasticsearch.disk.percent': ('gauge', 'disk_percent'),
 }
 
+TEMPLATE_METRICS = {
+    'elasticsearch.templates.count': ('gauge', 'templates', lambda templates: len(templates)),
```

Comment:
Just to be 100% sure: this really is a gauge, not a monotonic count, right? As in, the value at time `T_n` is in no way related to the value at `T_n-1`.

Reply:
Yes. The value is unrelated, it also won't vary too much, because templates are not created so often, but yeah.

Reply:
👍